packages/taquito-http-utils/src/taquito-http-utils.ts

```diff
@@ -3,23 +3,16 @@
  * @module @taquito/http-utils
  */
 
-import fetchAdapter from './fetch-adapter';
-import { STATUS_CODE } from './status_code';
-import axios from 'axios';
-import { HttpRequestFailed, HttpResponseError } from './errors';
+// const fetch = require('node-fetch');
 
-const isNode = typeof process !== 'undefined' && !!process?.versions?.node;
+import fetch from 'node-fetch';
```

Comment:
Wondering, in here, is this line (and the similar one in `packages/taquito-sapling/fetch-sapling-params.js`), specific to `node`? What about browser?

If we change this line so that it only imports if `fetch` is not already available?

From [this answer](https://stackoverflow.com/a/72440263):

```
let fetch = globalThis?.fetch;

if (!fetch && process?.versions?.node) {
  fetch = (await import('node-fetch')).default;
}

console.log(fetch);
```

But I guess something more relevant to us can conceptually be:
```
import fetch as node_fetch from 'node-fetch';
let fetch = globalThis?.fetch;
if (!fetch && process?.versions?.node) {
  fetch = node_fetch;
}
```

But I don't have enough context.

Reply:
the reason for that existing before is to use the `axios-fetch-adapter` when the runtime is a browser. 

But since we've now updated to fetch completely, there would be no need to check what runtime Taquito is currently running on.

Reply:
I mean, `fetch` is natively available on browsers, and `node-fetch` is there to make it available for `node`. So in browser we don't need to import `fetch` from `node-fetch`?

Reply:
oh I see what you mean. That makes sense, I hadn't consider that. I'll update it